apps/frontend/src/app/5_pages/LandingPage/components/ProtocolData/ProtocolData.tsx

```diff
@@ -0,0 +1,151 @@
+import React, { FC, useCallback, useRef, useState } from 'react';
+
+import axios, { Canceler } from 'axios';
+import { t } from 'i18next';
+
+import { Button, ButtonStyle } from '@sovryn/ui';
+
+import { AmountRenderer } from '../../../../2_molecules/AmountRenderer/AmountRenderer';
+import {
+  BITCOIN,
+  BTC_RENDER_PRECISION,
+  TOKEN_RENDER_PRECISION,
+  USD,
+} from '../../../../../constants/currencies';
+import { useInterval } from '../../../../../hooks/useInterval';
+import { translations } from '../../../../../locales/i18n';
+import {
+  DATA_REFRESH_INTERVAL,
+  DEFAULT_LOCKED_DATA,
+  DEFAULT_VOLUME_DATA,
+  LOCKED_DATA_URL,
+  VOLUME_DATA_URL,
+} from './ProtocolData.constants';
+
+const pageTranslations = translations.landingPage.protocolDataSection;
+
+export const ProtocolData: FC = () => {
+  const [lockedData, setLockedData] = useState(DEFAULT_LOCKED_DATA);
+  const cancelLockedDataRequest = useRef<Canceler>();
+
+  const [volumeData, setVolumeData] = useState(DEFAULT_VOLUME_DATA);
+  const cancelVolumeDataRequest = useRef<Canceler>();
+
+  const fetchLockedData = useCallback(() => {
```

Comment:
I was thinking about it but decided not to do it initially because it's a small component. But I can do it, it will be better for readability for sure.